packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ConnectorType } from '@logto/schemas';
+
+import { ConnectorMetadata } from '../types';
+
+export const metadata: ConnectorMetadata = {
+  id: 'aliyun-dm',
+  type: ConnectorType.SMS,
+  name: '阿里云邮件推送',
```

Comment:
this should be typed as l10n key or string, e.g. `name: keyof XX | string`

Reply:
`i18n` or `l10n` will be different in connector, I will implement it in a new PR, created LOG-1299 for tracking.